ParameterDataApi.yaml

```diff
@@ -0,0 +1,246 @@
+      required: 
+        - col0
+          
+    ColumnElementDefinition:
+      type: object
+      description: The column elements      
+      properties:
+        id:
+          description: The identifier for the column
+          type: string
+          format: byte
+        
+        name: 
+          description: The name of the column
+          type: string
+          
+        dataType:
+          $ref: '#/components/schemas/DataType'
+          
+        valueType:
+          $ref: '#/components/schemas/RenderType'          
+          
+        formatString:
+          description: The format specification for the value field in the column.
+          type: string
+          
+        aggregate:
+          description: The aggregate specification for the value field in the column.
+          type: string
+          
+      required:
+        - id
+        - name
+        - dataType
+        - valueType
+        - formatString
+        - aggregate
+
+    DataDefinition:
+      type: array
+      items: 
+        $ref: '#/components/schemas/DataElement'
+        
+    DataElement:
+      type: object
+      properties:
+        ts:
+          description: the timestamp of the sample
+          type: string
+          format: date-time
+            
+        f:
+          type: array
+          items:
+            $ref: '#/components/schemas/ValueSpec'
+            
+      required:
+        - ts
+        - f
+
+    DataType:
+      type: string
+      enum:
+        - NUMBER     
+        
+    HeaderDefinition:
+      type: object
+      properties: 
+        startTime:
+          description: The start time for the time series
+          type: string
+          format: date-time
+        
+        endTime: 
+          description: The end time for the time series
+          type: string
+          format: date-time
+        
+        recordCount: 
+          description: The number of records in the data.
+          type: integer
+          
+        columns:
+          $ref: '#/components/schemas/ColumnDefinition'
+        
+      required:
+        - startTime
+        - endTime
+        - recordCount
+        - columns
+
+    RenderType:
+      type: string
+      enum:
+        - VALUE
+
+    TimeSeriesDataDefinition:
+      type: object
+      description: A definition of time series data. This is based upon Json Time Series, Version 1.0. (https://docs.eagle.io/en/latest/reference/index.html)
+      properties:
+        docType:
+          type: string          
+          description: The type of the document
+
+        version:
+          type: string
+          description: The version string for the time series data definition
+
+        header:
+          type: object
+          properties:
+            header:
+              $ref: '#/components/schemas/HeaderDefinition'
+
+            data:
+              $ref: '#/components/schemas/DataDefinition'
+
+      required: 
+        - header
+        - data
+
+    ValueSpec:
```

Comment:
Readability vs data size - v, q, and a are pretty non-descript names for this. Presumably this is from the original spec, but depending on tolerance for deviation, this may be an opportunity for improvement